sonar-project.properties

```diff
@@ -4,5 +4,5 @@ sonar.sources=demo/,docs/,packages/,storybook/
 sonar.tests=packages/
 sonar.test.inclusions=packages/**/__tests__/**,packages/**/*.spec.ts
 sonar.exclusions=packages/**/__tests__/**,packages/**/*.spec.ts,storybook/**/*.stories.tsx,storybook/**/mockServiceWorker.js,storybook/**/mocks/**
-sonar.cpd.exclusions=packages/**/__tests__/**,packages/**/*.spec.ts,storybook/**/*.stories.tsx,storybook/**/mockServiceWorker.js,storybook/**/mocks/**
+sonar.cpd.exclusions=demo/site-pages/**,demo/site/**,packages/**/__tests__/**,packages/**/*.spec.ts,storybook/**/*.stories.tsx,storybook/**/mockServiceWorker.js,storybook/**/mocks/**
```

Comment:
it's not ideal, but i think its ok if we skip code duplication for the (mostly identical) demo projects (site and site-pages)

Reply:
what if we only skip duplication for demo/site-pages and keep detection of demo/site?

Reply:
that would be nice yes :+1:

Reply:
Fixed: 9e306d63f1d6f53c8c3e5a418d909f059b8d9439